spec/c08chalenges/non_overlapping_intervalSpec.js

```diff
@@ -0,0 +1,23 @@
+const insertInterval = require('../../src/c08challenges/non_overlapping_interval')
+
+describe('No overlapping intervals', function () {
+    it('Should find the correct position of non-overlaping intervals', function () {
```

Comment:
What position? The results look like it merges the newInterval with the given intervals. Give the test a clearer name.